src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__unit"></div>
```

Comment:
Pay attention to the task requirements, rename classes according to it
<img width="753" alt="Screenshot 2022-11-24 at 11 03 36" src="https://user-images.githubusercontent.com/104769168/203740434-3a982172-556c-4cb5-a1b0-7d0d47cf72a2.png">